doc/stages/filters.hexbin.md

```diff
@@ -131,7 +160,12 @@ preserve_topology
 
 smooth
 
-: Use GEOS simplify operations to smooth boundary to a tolerance \[Default: true\]
+: Use GEOS simplify operations to smooth boundary to a tolerance. Not compatible with H3 \[Default: true\]
```

Comment:
Maybe more accurate to say "not implemented" although it's probably not something we want to implement either. Let's get out of the "simplification" business.

Reply:
It might be a good idea to set a higher default precision for H3 for the same reasons I changed this. If people want to reference the output to other H3 data more accuracy is probably better

Reply:
If this is a management decision, that's fine, but I don't think there's any technical reason. I thought the reason to smooth was for looks. The same issue exists with H3. I don't really care, though.